Modix.Services/Promotions/PromotionsService.cs

```diff
@@ -197,10 +206,30 @@ public void ValidateComment(string content)
             }
         }
 
+        public async Task UpdateOrAddComment(long campaignId, PromotionSentiment sentiment, string content)
```

Comment:
Go ahead and make an issue to refactor `.AddComment()` and `UpdateComment()` into this method, and remove them.